auth/oidc.go

```diff
@@ -95,6 +95,19 @@ type OIDCProvider struct {
 	DiscoveryURI            string   `json:"discovery_url,omitempty"`          // Non-standard discovery endpoints
 	DisableConfigValidation bool     `json:"disable_cfg_validation,omitempty"` // Bypasses config validation based on the OIDC spec.  Required for some OPs that don't strictly adhere to spec (eg. Yahoo)
 
+	// DisableCallbackState determines whether or not to maintain state between the
+	// Authentication (OAuth 2.0 Authorization) request and the callback. The default
+	// value of DisableCallbackState is false, which means state is maintained between
+	// auth request and the callback. It can be disabled through provider configuration
+	// by setting property "disable_callback_state": false. Disabling callback state is
```

Comment:
```suggestion
	// by setting property "disable_callback_state": true. Disabling callback state is
```

Reply:
This change has been revised in latest revision.